README.md

```diff
@@ -89,12 +89,10 @@ Prerequisites
 
 	CentOS:
 	```
-	sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-	sudo yum install -y http://downloads.globus.org/toolkit/gt6/stable/installers/repo/rpm/globus-toolkit-repo-latest.noarch.rpm
-	sudo yum-config-manager --enable Globus-Connect-Server-5-Stable
-	sudo yum-config-manager --enable Globus-Toolkit-6-Stable
+	wget -q https://downloads.globus.org/globus-connect-server/stable/installers/repo/rpm/globus-repo-latest.noarch.rpm
+	sudo rpm --force -i globus-repo-latest.noarch.rpm
 	sudo yum install -y globus-gridftp-server-progs globus-gass-copy-progs
-	sudo yum install -y globus-common-devel globus-gridftp-server-devel globus-gridmap-callout-error-devel
+	sudo yum --disablerepo epel install -y globus-common-devel globus-gridftp-server-devel globus-gridmap-callout-error-devel
 	sudo yum install -y libcurl-devel
 	sudo yum install -y git
 	sudo yum install -y gcc-c++
```

Comment:
could combine these...

`sudo yum install -y libcurl-devel git gcc-c++`

and could add `make` to the end - it wasn't on the vanilla container...